spacebrew.js

```diff
@@ -20,7 +20,13 @@ var path = require('path')
     , serveStatic = require('serve-static')
     , http = require('http')
     , finalhandler = require('finalhandler')
+    , AJV = require('ajv')
+    , fs = require('fs')
     ;
+
+//setup validator
+var ajv = AJV();
+var validate = ajv.compile(JSON.parse(fs.readFileSync('schema.json')));
```

Comment:
hmmm, should these be inside spacebrew.createServer, so they are scoped instead of global? That probably isn't an issue with Node.js though right?